packages/bigtable/package.json

```diff
@@ -50,12 +50,12 @@
     "bigtable"
   ],
   "dependencies": {
-    "@google-cloud/common": "^0.12.0",
-    "@google-cloud/common-grpc": "^0.2.1",
+    "@google-cloud/common": "^0.13.0",
+    "@google-cloud/common-grpc": "^0.3.0",
     "arrify": "^1.0.0",
     "concat-stream": "^1.5.0",
     "create-error-class": "^3.0.2",
-    "dot-prop": "^3.0.0",
+    "dot-prop": "^4.1.1",
```

Comment:
This might be a problem-- I think they switched from callbacks to promises, so we'd probably need some code changes, but I suppose CI will answer that for us.